LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Monzo
```

Comment:
Does this need to be more specific? e.g. Monzo Bank Ltd?

Reply:
yh, probably worthwhile just in case